MAUI/DataForm/Editors.md

```diff
@@ -0,0 +1,1069 @@
+---
+layout: post
+title: Editors in Maui DataForm control | Syncfusion
+description: Learn here all about Editors support in Syncfusion Maui DataForm (SfDataForm) control, its elements and more.
+platform: Maui
+control: SfDataForm
+documentation: UG
+---
+
+# Editors in Maui DataForm (SfDataForm)
+
+The data form supports several built-in editors as follows:
+
+<table>
+<tr>
+<th>Editor name</th>
+<th>Editor class</th>
+<th>Supported Data Type/Attribute</th>
+<th>Input control loaded</th>
+</tr>
+<tr>
+<td>
+Text
+</td>
+<td>
+{{'[DataFormTextEditor
+</td>
+<td>
+The String type property and any other type apart from the below specified cases.
+</td>
+<td>
+Entry
```

Comment:
Add links for all editor views
Entry
Editor
CheckBox,
Switch
DatePicker,
TimePicker,
SfComboBox
SfAutoComplete
RadioButton,
Switch

Reply:
link added for above mentioned editors